devel/gdub/Portfile

```diff
@@ -3,7 +3,8 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        dougborg gdub 0.1.0 v
+github.setup        dougborg gdub ebe14f110153f05f3a50e7ff2eaa8d2704cde9d9
+version             0.1.0.1
```

Comment:
Small nitpick, but I would suggest adding a short comment here, just so anyone in the future looking at this port would understand what has been done w.r.t. the git commit and version.

Reply:
@cjones051073 , thanks for the clarifications, however I find it really sad we both (+the ones who are watching the repo) spend time on such a minor issues.